addons/mod_tool/global/store.gd

```diff
@@ -15,13 +15,17 @@ var editor_file_system: EditorFileSystem
 var error_color := ""
 
 var name_mod_dir := "" setget set_name_mod_dir
+var mod_name := ""
+var mod_namespace := ""
```

Comment:
shouldn't these two be covered by the manifest?

Reply:
yeah that could work

Reply:
*fixed [8afb088](https://github.com/GodotModding/godot-mod-tool/pull/45/commits/8afb088ca1026a3afa21bda3b09c538d5c55c117)*